src/Control/Lens/Fold.hs

```diff
@@ -424,8 +424,8 @@ takingWhile p l pafb = fmap runMagma . traverse (cosieve pafb) . runTakingWhile
 --
 -- Note: Many uses of this combinator will yield something that meets the types, but not the laws of a valid
 -- 'Traversal' or 'IndexedTraversal'. The 'Traversal' and 'IndexedTraversal' laws are only satisfied if the
--- new values you assign also pass the predicate! Otherwise subsequent traversals will visit fewer elements
--- and 'Traversal' fusion is not sound.
+-- new value you assign to the first target also does not pass the predicate! Otherwise subsequent traversals
```

Comment:
The reason why you must not change the targets in a way that changes the predicate is that it would change the elements you traverse over. But as long as the first element for which the predicate fails stays the same the elements traversed over stay the same.